fixed issued in csv2libsvm where `-l -1` would yield TypeError

stevenpollack opened the pull request:
* line 51 set label to 1, when it should've set it to "1"

zygmuntz commented:
Thanks, fixed this (not using your PR because you introduced spaces in line 13).

stevenpollack commented:
No worries; happy to have it fixed regardless!

On Sun, Sep 20, 2015 at 11:23 PM, Zygmunt Zając <notifications@github.com>
wrote:

> Thanks, fixed this (not using your PR because you introduced spaces in
> line 13).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zygmuntz/phraug2/pull/9#issuecomment-141834941>.
>



-- 

~~~~~~~
Please note that I've moved my primary email to steven@gnobel.com.
